cli/scorecard/cmd.go

```diff
@@ -21,6 +23,9 @@ func init() {
 	Cmd.Flags().StringVar(&flags.policyPath, "policy-path", "", "Path to directory containing validation policies")
 	Cmd.MarkFlagRequired("policy-path")
 
+	Cmd.Flags().StringVar(&flags.outputPath, "output-path", "", "Path to directory to contain scorecard outputs. Output to console if not specified")
+	Cmd.Flags().StringVar(&flags.outputFormat, "output-format", "", "Format of scorecard outputs, can be txt, json or csv, default is txt")
```

Comment:
Could we handle this flag using a viper flag + default? https://github.com/spf13/viper#establishing-defaults

Reply:
addressed in e4926b7